StyleGuide.md

```diff
@@ -0,0 +1,37 @@
+# RetroConnect Team Style Guide
+---
+
+## Areas of Focus:
+---
+* **External Documentation**
+* **Class Documentation**
+* **Interal Documentation & Naming**
+* **Indentation**
+* **Miscellaneous**
+---
+#### External Documentation:
+* At the top of each file created, the name of the team member working on this file and a brief description of the purpose of the file should be added.
+#### Class Documentation:
+* Each support class is to be preceded with a block comment that includes:
```

Comment:
We should include an example of what the block comment looks like.  
We should also specify what documentation style we are using, e.g. Doxygen.